contentcuration/contentcuration/frontend/shared/views/channel/ChannelModal.vue

```diff
@@ -134,6 +134,7 @@
         changed: false,
         showUnsavedDialog: false,
         diffTracker: {},
+        dialog: true,
```

Comment:
Temporarily emulate a usual non-dialog page behavior (until we can update these modal components to regular pages)